explorer/testdata/destructor/call_destructor_from_destructor.carbon

```diff
@@ -30,15 +32,16 @@ class B{
         Print("DESTRUCTOR B {0}",me.n);
     }
     fn Create(x: i32) -> B{
-       return {.n = x, .a = A.Create(3) };
+       return {.n = x, .a1 = A.Create(4),.a2 = A.Create(3) };
```

Comment:
This test changes doesn't look closely related to the rest, am I missing something?

Reply:
It is tested that more than one member of the object is destroyed